apps/cli/src/constants.ts

```diff
@@ -1,6 +1,24 @@
 import { join } from "node:path";
 import envPaths from "env-paths";
 
-const APP_CONFIG = envPaths("webstudio").config;
+const GLOBA_APP_CONFIG = envPaths("webstudio").config;
 const GLOBAL_CONFIG_FILE_NAME = "webstudio-config.json";
-export const GLOBAL_CONFIG_FILE = join(APP_CONFIG, GLOBAL_CONFIG_FILE_NAME);
+export const GLOBAL_CONFIG_FILE = join(
+  GLOBA_APP_CONFIG,
+  GLOBAL_CONFIG_FILE_NAME
+);
+
+export const LOCAL_APP_NAME = "webstudio-project";
+export const LOCAL_APP_CONFIG_FILE_NAME = "webstudio.json";
+export const LOCAL_CONFIG_FILE = join(
+  process.cwd(),
+  LOCAL_APP_NAME,
+  LOCAL_APP_CONFIG_FILE_NAME
+);
```

Comment:
Let's expose only resolved paths and have separate names for json with data and json with config.

Reply:
You can use `join(cwd(), webstudio.json)` here too. cwd does not change if you not call proces.chdir